ij/src/main/kotlin/org/jetbrains/academy/test/system/test/BaseIjTestClass.kt

```diff
@@ -1,8 +1,55 @@
 package org.jetbrains.academy.test.system.test
 
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiRecursiveElementWalkingVisitor
 import com.intellij.testFramework.fixtures.BasePlatformTestCase
+import org.jetbrains.kotlin.psi.KtBlockExpression
+import org.jetbrains.kotlin.psi.KtNamedFunction
+import org.jetbrains.kotlin.psi.KtProperty
 
 /**
  * A Base test class to create tests with PSI without adding intellij dependency into courses directly.
  */
-open class BaseIjTestClass : BasePlatformTestCase()
+open class BaseIjTestClass : BasePlatformTestCase() {
```

Comment:
Probably you need to create a new class with a specific goal that implements BaseIjTestClass, isn't? Because it is impossible to add everything to BaseIjTestClass class